### ISSUE TITLE
can't compile bbb-apps-akka 

### ISSUE THREAD

**Issue by saxihuangxing** [bigbluebutton/bigbluebutton#7838]
when i compile commit build.sh in bbb-papps-akka , it  aways stuck with resolving Dependencies.... , the issue is for  the last step  grails clean.  i wait one day and it still show that,i have tried  many times and still have no idea about the issue.  i am sure my internet is ok , my college in same env don't have this issue , we only in different VM. i have compare our VM but have find the reason. 
my enviroment is ubuntu16.04 ,gradle 5.1.1,grails 3.3.9. 
thanks for give me some advise , i have spend one week to solve this issue and still get no idea about it.  i am not familly with gradle and grails.

**Comment by capilkey** [comment]
Can you post the full logs from the build, please.

**Comment by saxihuangxing** [comment]
HI：this is the log when i run build.sh , but it's just get a little informations , i don't know if it can help. 


Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 1s
1 actionable task: 1 executed

Deprecated Gradle features were used in this build, making it incompatible with Gradle 6.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/5.1.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD SUCCESSFUL in 0s
1 actionable task: 1 up-to-date
[1A[1D[0K[1;33m| [22;39mResolving Dependencies. Please wait...[m

**Comment by saxihuangxing** [comment]
@capilkey

**Comment by GhaziTriki** [comment]
@saxihuangxing I am the developer responisble of upgrading bbb-web. The build is successful as I see. However are you trying to run your compiled code for production?

**Comment by saxihuangxing** [comment]
@GhaziTriki   yes , i am trying to run bbb-web . but i meet this issue and take a lot time and don't know the reason.  my collegues compile this don't have this issues too.  i don't know what's the reason. i have compare the env with my collegue, we have use the same gradle , grails ,maven version

**Comment by GhaziTriki** [comment]
@saxihuangxing If you do `sudo service bbb-web stop` then `./run.sh` from your `bigbluebutton-web` directory, does your compiled bbb-web start?

**Comment by saxihuangxing** [comment]
hi , 
 when i run build.sh , it stuck with the last step grails clean with resolving dependencies.... , i can't get any more infos , so i don't know how to resolving this probleam .  can you add my skype  1229958344



------------------ 原始邮件 ------------------
发件人: "Ghazi Triki"<notifications@github.com>;
发送时间: 2019年7月30日(星期二) 下午2:21
收件人: "bigbluebutton/bigbluebutton"<bigbluebutton@noreply.github.com>;
抄送: "黄新"<1229958344@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [bigbluebutton/bigbluebutton] can't compile bbb-web (#7838)




@saxihuangxing If you do sudo service bbb-web stop then ./run.sh from your bigbluebutton-web directory, does your compiled bbb-web start?

**Comment by GhaziTriki** [comment]
@saxihuangxing We've update the docs for development of 2.2. You will find the right tools versions and the procedure to compile the source code.
Please check starting from the following section http://docs.bigbluebutton.org/2.2/dev.html#setup-a-development-environment

**Comment by GhaziTriki** [comment]
@saxihuangxing More updates on docs http://docs.bigbluebutton.org/2.2/dev.html#developing-bbb-web, we added how to deploy for production. We tested all the instructions. We didn't find any issue.
